templates/base.html

```diff
@@ -105,7 +105,12 @@ <h5 class="white-text">Links</h5>
           <ul>
             <li><a href="https://t.me/brasil_io">
                 Nosso Canal no Telegram
-                <img style="height: 1.2em" src="{% static 'img/logo-telegram.png' %}" alt="Logo Telgram"></a></li>
+                <img style="height: 1.2em" src="{% static 'img/logo-telegram.png' %}" alt="Logo Telgram"></a>
+            </li>
+            <li><a href="https://twitter.com/brasil_io">
+                Nosso Twitter
+                <img style="height: 1.2em" src="{% static 'img/logo-twitter.png' %}" alt="'Logo Twitter'"></a>
+            </li>  
```

Comment:
@Riverfount tudo parece ok. Você poderia só colocar um print do seu ambiente local para mostrar como esse link está aparecendo no frontend?

Reply:
Ok, to fazendo isso já!